autobahn/src/main/java/io/crossbar/autobahn/wamp/types/CallResult.java

```diff
@@ -0,0 +1,22 @@
+package io.crossbar.autobahn.wamp.types;
+
+import java.util.List;
+import java.util.Map;
+
+public class CallResult {
+    public List<Object> results;
+    public Map<String, Object> kwresults;
+
```

Comment:
copy constructor would take another CallResult and create a new CallResult with the values deep copied (not merely copying the references) - it's a minor point. we can add that later if needed.